src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/IncrementalBuildTest.cs

```diff
@@ -501,6 +501,7 @@ public void LibraryProjectTargetsDoNotBreak ()
 					Assert.IsFalse (b.Output.IsTargetSkipped (target), $"`{target}` should *not* be skipped!");
 				}
 
+				System.Threading.Thread.Sleep (1000);
```

Comment:
maybe use the `BaseTest.WaitFor` static method for this. We use that allot for the DebuggingTests.